conda/gateways/repodata/__init__.py

```diff
@@ -57,14 +55,21 @@ class Response304ContentUnchanged(Exception):
 
 
 class CondaRepoInterface(RepoInterface):
-    def __init__(self, url: str, repodata_fn: str | None, **kwargs) -> None:
-        self._url = url
-        self._repodata_fn = repodata_fn
+    """
+    Provides an interface for retrieving repodata data from channels
+    """
 
-        self._log = logging.getLogger(__name__)
-        self._stderrlog = logging.getLogger("conda.stderrlog")
+    #: Channel URL
+    _url: str
 
-    def repodata(self, state: dict) -> str | None:
+    #: Filename of the repodata file; defaults to value of conda.base.constants.REPODATA_FN
```

Comment:
does sphinx pick up the #: on the previous line; when I was documenting conda-package-handling I had trouble getting this to work https://github.com/conda/conda-package-handling/blob/main/src/conda_package_handling/api.py#L31

Reply:
I've never actually tried run it through Sphinx 🤔. TBH, I've always gone with a doc string directly below the variable name, but recently I was advised to start using this format...

Regardless, this isn't actually documented anywhere in any style guide, so it doesn't matter. It would be very nice to have a documenting style guide for this project though.

Reply:
I know the `#:` syntax will work for global vars and class vars.

[![Sphinx autodoc docs](https://user-images.githubusercontent.com/4546435/201954052-356daf86-0a93-4221-95e2-d4352f806e69.png)](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)

I don't think it's possible (or desirable) to document local variables so a plain `#` is best here.